### PR TITLE
AP '17 Chapter 3

### PR DESCRIPTION
**Pull request by sean-walker**
Added:
RNG	
Fifteen (1, 2, 3: merged into one)	
Analyze This
Sort Race
Seek
Scramble (1)
Simulate!

Breakout excluded for now pending updated version.

### CODE REVIEW

File: 2017/ap/problems/race/race.adoc [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+
+Incidentally, because it is not a so-called "target" specified in the `Makefile`, if when working on this problem you (inadvertently or intentionally) try to
+
+[source]
+----
+make helpers
+----
+
+you'll actually default to using the standard `Makefile` included with CS50 IDEfootnote:[The existence of this file is why your programs in Units 1 and 2 compiled with no trouble despite not having a `Makefile` in the directory.] which will just try to compile the `helpers.c` file alone into its own program. Problem is, if you open up `helpers.c`, there's no `main` function, so you'll probably get a whole bunch of cryptic error messages concluding with one along these lines:
```
Comment (Erin-c) [inline 2017/ap/problems/race/race.adoc]: replace `your programs in Units 1 and 2` with `your previous programs`

File: 2017/ap/problems/analyze/analyze.adoc [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+In 500-1,000 words, prepare an essay on your experience, expectations, growths, triumphs, and struggles up through this point in the course. You can spend as much time as you feel is appropriate on each of these questions, but over the course of your writing you should touch on at least the following topics:
+
+* Discuss your initial expectations of this course, and about how the course has either aligned with or deviated from these expectations.
+* Write about at least one programming problem and at least one writing problem that you enjoyed or did not enjoy. What specifically about that problem did you enjoy or not enjoy, and why do you feel that way?
+* Explain how your habits as a computer scientist have evolved. How do you do things differently at this point in the course from how you did the same things in Unit 1? For instance, has your debugging strategy changed? Do you find yourself more intuitively counting from 0 instead of 1? Do you find that some of what you've learned in this course has extended to other domains of interest to you? How?
```
Comment (Erin-c) [inline 2017/ap/problems/analyze/analyze.adoc]: Replace with `Chapter 1`

File: 2017/ap/problems/race/race.adoc [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+bubble sort benchmark:         52.032 seconds
+selection sort benchmark:      28.944 seconds
+insertion sort benchmark:      8.808 seconds
+----
+
+Ouch! Or lastly
+
+[source,subs=quotes]
+----
+~/workspace/unit3/race/ $ [underline]#./race -s 20000#
```
Comment (Erin-c) [inline 2017/ap/problems/race/race.adoc]: replace `unit` with `chapter`

**Comment by sean-walker** [comment]
@Erin-c e9f111f3c4e1b078b21a9f6118ee9e7e2c326449 fixed them, no?

**Pull request merged**